corehq/apps/userreports/tasks.py

```diff
@@ -104,7 +104,12 @@ def _iteratively_build_table(config, last_id=None):
     start_key = None
     if last_id:
         last_doc = _DOC_TYPE_MAPPING[config.referenced_doc_type].get(last_id)
-        start_key = [config.domain, config.referenced_doc_type, json_format_datetime(last_doc.recieved_on)]
+        date = None
+        if config.referenced_doc_type in _DATE_MAP.keys():
+            date = json_format_datetime(last_doc[_DATE_MAP[config.referenced_doc_type]])
+        start_key = [config.domain, config.referenced_doc_type]
+        if date:
```

Comment:
nit: this could be slightly easier to see what's going on if it was just one if statement:
```
if config.referenced_doc_type in _DATE_MAP.keys():
  date = json_format_datetime(last_doc[_DATE_MAP[config.referenced_doc_type]])
  start_key.append(date)
```
(unless json_format_datetime can return None which is doesn't look like it can)